### PR TITLE
Revise labels

### PR DESCRIPTION
**Pull request by frankharkins**
### Summary

* Added new labels
  * `Visualization`
  * `Productivity`
* Changed / merged existing labels
  * `plugin` -> `Transpiler plugin` to closer fits the description in `labels.json`.
    I've also removed this label from a few projects as I believe they added it thinking it meant "general Qiskit plugin"
  * `notebook` -> `Jupyter notebook`
  * Absorbed `convert` into `Converter`
  * Absorbed `simulation` into `Physics`
* Improved style / readability
  * Expanded acronyms
  * Replaced hypens with spaces
  * Use sentence-style capitalization
* Repo maintenance
  * Replaced references to "tags"
  * Synced label names across issue template, labels.json, and TOML files
  * Reviewed all members to check labels are appropriate

### CODE REVIEW

**Comment by javabster** [comment]
I think we said in the meeting we should keep "circuit simulator" and remove "simulation" correct? And any projects that are in the simulation category should be put into the physics or chemistry

**Comment by frankharkins** [comment]
@javabster thanks, done!

File: .github/ISSUE_TEMPLATE/submission.yml [hunk 1]
```diff
@@ -69,31 +69,41 @@ body:
...
-        - finance
-        - game
-        - notebook
-        - converter
+        - Advocate mentorship project
+        - Algorithms
+        - Chemistry
+        - Circuit simulator
+        - Circuit
+        - Cloud
```
Comment (frankharkins) [inline .github/ISSUE_TEMPLATE/submission.yml:85]:
```suggestion
```

**Comment by frankharkins** [comment]
Merging as this was discussed with the team last meeting

**Pull request merged**